src/sagemaker/experiments/_environment.py

```diff
@@ -53,9 +54,9 @@ def __init__(self, environment_type, source_arn):
     @classmethod
     def load(
         cls,
-        training_job_arn_env=TRAINING_JOB_ARN_ENV,
-        processing_job_config_path=PROCESSING_JOB_CONFIG_PATH,
-        transform_job_batch_var=TRANSFORM_JOB_ENV_BATCH_VAR,
+        training_job_arn_env: str = TRAINING_JOB_ARN_ENV,
+        processing_job_config_path: str = PROCESSING_JOB_CONFIG_PATH,
+        transform_job_arn_env: str = TRANSFORM_JOB_ARN_ENV,
```

Comment:
i dont understand why these are args in the first place - they will never change and so should be constants

Reply:
This was kept when migrating the Experiment SDK to main SDK: https://github.com/aws/sagemaker-experiments/blob/caf7b147ddc6fe44f40b07ca3a5e011d9c57cfae/src/smexperiments/_environment.py#L47.
In Exp SDK, the `TrialComponentEnvironment.load` is a public facing method, I guess leaving these local vars may be useful when calling/testing it in different env.
However, as given it's internal method in our main SDK use case, agree that these local args are not necessary. Will remove them

Reply:
This [unit test](https://github.com/aws/sagemaker-python-sdk/blob/a94a3b17fb7f3aa46906869cba0a24cb5522d4d2/tests/unit/sagemaker/experiments/test_environment.py#L58) migrated from Exp SDK seems prove my assumption.
With these args, they make this method to be more flexible and much easier to test in terms of mocking the processing job env (and the transform job env in the future as well once the transform team gets the config file ready for us).
Thus, after a second thought, I'll keep this behavior.